assets/src/edit-story/elements/video/index.js

```diff
@@ -46,11 +46,12 @@ export const defaultAttributes = {
     posterId: null,
     poster: null,
     videoId: 0,
+    title: '',
   },
 };
 
 export const panels = [
   ...MEDIA_PANELS,
-  PanelTypes.VIDEO_POSTER,
+  PanelTypes.VIDEO_ACCESSIBILITY,
```

Comment:
According to Figma, the accessibility panel should come after the link panel.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/652/commits/eef36fbe3357f8343b08bf9f4daa513550c45378